core/tests/protractor_desktop/additionalPlayerFeatures.js

```diff
@@ -238,7 +252,6 @@ describe('Full exploration editor', function() {
     await creatorDashboardPage.get();
     await creatorDashboardPage.clickCreateActivityButton();
     await creatorDashboardPage.clickCreateExplorationButton();
-    await explorationEditorMainTab.exitTutorial();
```

Comment:
Why is this removed?

Reply:
Welcome Modal is not expected over here, hence no need to exit tutorial.